documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Datasets can change over time. Instead of hard-coding numeric values, use code to print out the specifics. This is mainly for the "891 entries" part. It's likely too difficult or awkward to avoid talking about the different columns.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>